Assets/LeapMotionModules/ElementRenderer/Scripts/Renderers/LeapGuiBakedRenderer.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+using Leap.Unity;
+using Leap.Unity.Query;
+
+[AddComponentMenu("")]
+[LeapGuiTag("Baked")]
+public class LeapGuiBakedRenderer : LeapGuiMesherBase {
+
+  #region INSPECTOR FIELDS
+
+  [SerializeField]
+  private SingleLayer _layer = 0;
+
+  [SerializeField]
+  private MotionType _motionType = MotionType.Translation;
+  #endregion
+
+  #region PRIVATE VARIABLES
+
+  //## Rect space
+  private const string RECT_POSITIONS = LeapGui.PROPERTY_PREFIX + "Rect_ElementPositions";
+  private List<Vector4> _rect_elementPositions = new List<Vector4>();
+
+  //## Cylindrical/Spherical spaces
+  private const string CURVED_PARAMETERS = LeapGui.PROPERTY_PREFIX + "Curved_ElementParameters";
+  private List<Vector4> _curved_elementParameters = new List<Vector4>();
+
+  #endregion
+
+  protected override void OnValidate() {
+    base.OnValidate();
+
+    //Currently full motion is not supported
+    if (_motionType == MotionType.Full) {
+      _motionType = MotionType.Translation;
```

Comment:
That makes sense to me, I'll go ahead and remove it from the dropdown for now, there already is a ticket for it.